lib/clientBuilder.js

```diff
@@ -244,16 +245,17 @@ proto.fingerprint = function(options) {
     }
     if (msg.type === 'FingerprintServiceReady') {
       if (iframe.contentWindow) {
-        iframe.contentWindow.postMessage(JSON.stringify({
-          type: 'GetFingerprint'
-        }), sdk.options.url);
+        iframe.addEventListener('load', function() {
```

Comment:
is this the actual fix or is it a combination of this `load` event listener and the order of configuring and attaching the iframe?